### DISCUSSION TITLE
Can we add a map of content at the mainpage?

### DISCUSSION THREAD

**Discussion by jesili** [outline/outline#7392] [Ideas]
Adding a map of content or a mind map can make the structure of documents clearly. When there are many documents, a beginner will feel confuse and don't know where to start. I found a mind map tool called markmap that can render the markdown to the mindmap. I'm not sure if it can be integrated to  outline. The link is here : [markmap](https://markmap.js.org/).

**Comment by wenqvip** [comment]
Support! I want this feature in some cases like conclude a article or a course, just input the outlines and the mind-map appears.
